content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -5,9 +5,8 @@ metaDescription: Prisma Client features that are currently in preview.
 
 <TopBlock>
 
-The following [Preview](/about/prisma/releases#preview) feature flags are available for Prisma Client and the Prisma schema:
+The following [preview](/about/prisma/releases#preview) feature flags are available for Prisma Client and the Prisma schema:
```

Comment:
Did we change the rules around capitalization here? "Preview feature" was a "term" before.

Reply:
In the past we always uppercased these.